domains/manhha.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "haconbk"
+    },
+    "record": {
+        "CNAME": "https://haconbk.github.io/"
```

`CNAME` record must contain a domain only, remove `https://` and `/`
```suggestion
        "CNAME": "haconbk.github.io"
```